modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/LanguageServiceEndpoint.java

```diff
@@ -65,6 +72,22 @@ void setLanguageService(LanguageService languageSrv) {
     this.languageSrv = languageSrv;
   }
 
+  private static Set<String> excludedLocales;
+
+  public static final String EXCLUDE_CONFIG_KEY = "org.opencastproject.adminui.languages.exclude";
+
+  /** OSGi callback if properties file is present */
+  @Override
+  public void updated(Dictionary<String, ?> properties) throws ConfigurationException {
+    if (properties == null) {
+      logger.info("No configuration available, using defaults");
+      return;
+    }
+
+    String excludes = StringUtils.trimToEmpty((String) properties.get(EXCLUDE_CONFIG_KEY));
+    excludedLocales = new HashSet<>(Arrays.asList(excludes.split(",")));
```

Comment:
This would cause problems when the value contains whitespaces like this: `prop = de, en, fr`.
Maybe use something like:

```
excludedLocales.addAll(StringUtils.split(properties.get(EXCLUDE_CONFIG_KEY).toString(), ", "));
```

This would also split `en fr` but that should not cause a problem.

Reply:
IMO there are going to be few places in the world where this is going to be a useful feature, so I'm not going to worry terribly about edge cases like this.